src/index.html

```diff
@@ -22,6 +22,347 @@
+        <a href="#">
+          <img
+            class="img"
+            src="/src/images/logo.png"
+            alt="MOYO logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                class="nav__link is-active"
+                href="#"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Samsung
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Smartphones
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                data-qa="nav-hover"
+                href="#"
+              >
+                Gadgets
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Tablets
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Photo
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+    </div>
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="/src/images/imac.jpeg"
+          alt="Product Image"
+          class="card__image"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="card__stars card__stars--active-4">
+            <div class="card__stars-star"></div>
+            <div class="card__stars-star"></div>
+            <div class="card__stars-star"></div>
+            <div class="card__stars-star"></div>
+            <div class="card__stars-star"></div>
+          </div>
+          <span class="card__reviews-amount">Reviews: 5</span>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="card__button"
+          data-qa="card-hover"
```

Comment:
The `data-qa` attribute for the 'Buy' link inside the first card should be `card-hover` as specified in the task requirements.